test/screenshot/spec/mdc-menu/classes/baseline.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2018 Google Inc.
```

Comment:
Nit: 2019 😀

Reply:
Done.

I should've anticipated this 😜😅